spec/javascripts/components/notes/note_spec.js

```diff
@@ -1,59 +1,69 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render } from "@testing-library/react";
+import { screen } from '@testing-library/dom';
+import userEvent from "@testing-library/user-event";
 import NoteComponent from 'components/notes/Note';
 import Note from 'models/note.js';
 
-describe('<Note />', function() {
-  let note;
+window.document.getSelection = jest.fn()
```

Comment:
Can we do and restore it in a beforeEach hook instead as we do for `I1n8.t` and `md.makeHtml`?